bots/jabber_mirror_backend.py

```diff
@@ -132,6 +142,7 @@ def leave_muc(self, room):
         self.plugin['xep_0045'].leaveMUC(muc_jid, self.nick)
 
     def message(self, msg):
+        # type: (Any) -> Any
```

Comment:
Hm... looks strage the implementation should be
```
test_execution()
return
``` then...
so ok `Any` looks correct ;-( @pweaver, @jdherg  Do you have any ideas how to do it better?